cfnstack/assets.go

```diff
@@ -117,23 +119,33 @@ type Asset struct {
 
 type AssetLocationProvider struct {
 	s3URI     string
+	s3Region  string
 	stackName string
 }
 
 type AssetLocation struct {
-	ID     AssetID
-	Key    string
-	Bucket string
-	Path   string
+	ID       AssetID
+	Key      string
+	Bucket   string
+	Path     string
+	S3Region string
```

Comment:
This should be `Region Region` rather than `S3Region string` once https://github.com/coreos/kube-aws/pull/390/files#r104814373 is implemented.